### ISSUE TITLE
v1.0.2 ECID not accepted

### ISSUE THREAD

**Issue by Critter** [CokePokes/PhantomExo#14]
Even with removing the trailing zeros, the ECID entry screen just keeps popping back up when I hit submit. 

I only receive an error when I hit cancel. Then it says Invalid ECID...

**Comment by Critter** [comment]
I just tried again with 1.0.2-1. Now it is telling me that the ECID is not valid instead of just flashing back to the popup.

**Comment by CokePokes** [comment]
In v1.0.2-1 you must include leading 0's. A total of 16 chars should be entered.

**Comment by Critter** [comment]
Gotcha. I was removing the 0's.

That said...

I made it through the entire process. udid, wifi/bluetooth. then it popped up a message stating that the ```supplied info did not match given UDID: xxxxx (different from my actual UDID) ECID: xxxxx (different from my actual ECID)```

**Comment by instaglen** [comment]
Getting the same error as critter above

**Comment by Critter** [comment]
I wonder if the issue is arising from me having had purchased Phantom before using the same email address and the server side component is pulling up the old information?

**Comment by Bvnao** [comment]
I have put an space before and after the ECID, it work but the last step wont work.. "Supplied Info did not match.."

**Comment by Critter** [comment]
Just went through the entire process on 1.0.3 - Still receiving the mismatch error at the end.

**Comment by instaglen** [comment]
Same here. 

Just bought the license yesterday for my old iPhone 6 and my iPhone 7. 

iPhone 6 worked fine till now (installed phantom via Cydia).

For the iPhone 7, when I registered it yesterday it asked for ECID without the 000 at the front. I entered as accordingly and got registered. However, as soon as I updated and reinstalled the Phantom sideloaded version 1.0.3 I can't seem to register with my ECID
